packages/hooks/src/test/index.test.js

```diff
@@ -740,3 +740,40 @@ test( 'add multiple all actions and run it any hook to trigger them by priority'
 	expect( window.actionValue ).toBe( 'ba' );
 } );
 
+test( 'checking hasAction and hasFilter with named callbacks', () => {
+	// hasAction tests
+	addAction( 'test.action', 'my_callback', () => {} );
+	expect( hasAction( 'test.action', 'not_my_callback' ) ).toBe( false );
+	expect( hasAction( 'test.action', 'my_callback' ) ).toBe( true );
+
+	// test removing
+	removeAction( 'test.action', 'my_callback' );
+	expect( hasAction( 'test.action', 'my_callback' ) ).toBe( false );
+
+	// test removeAll
```

Comment:
This looks like it can be a start on its own `test`.

Reply:
👍